src/Language/PureScript/AST/Exported.hs

```diff
@@ -144,3 +141,16 @@ isDctorExported ident (Just exps) ctor = test `any` exps
   test (TypeRef _ ident' Nothing) = ident == ident'
   test (TypeRef _ ident' (Just ctors)) = ident == ident' && ctor `elem` ctors
   test _ = False
+
```

Comment:
I don't see how we would, that information is gone by then. Also this code isn't used in the normal compilation process; just by docs and the repl (and the repl should be using Environment instead anyway, this is mistake I made ages ago)